.github/workflows/tests.yml

```diff
@@ -7,23 +7,24 @@ on:
 jobs:
   rake:
     runs-on: "ubuntu-latest"
-    continue-on-error: true
     strategy:
       fail-fast: false
       matrix:
         ruby_version:
-          - "2.3.8"
-          - "2.4.5"
-          - "2.5.3"
-          - "2.6.3"
-          - "2.7.2"
+          - "2.3"
+          - "2.4"
+          - "2.5"
+          - "2.6"
+          - "2.7"
```

Comment:
Use the latest patch release available on the CI server.

Reply:
👍